weather-app/src/app/app.component.ts

```diff
@@ -1,38 +1,33 @@
 import { Component, OnInit } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { API_APP_ID, API_WEATHER_ENDPOINT } from './utils/constants';
+import { CurrentWeatherService } from './current-weather.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
+  providers: [CurrentWeatherService]
```

Comment:
We can probably remove this providers field, since the service already has Injector decorator.

Reply:
Thank you, I have deleted line 10